packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -881,6 +881,7 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 	public async getDataStore(
```

Comment:
Unrelated to your change but this is not used outside this file so it can be private.